.circleci/config.yml

```diff
@@ -269,88 +269,86 @@ workflows:
   docs:
     jobs:
       - docs
-      - docs_approve_deploy:
-          type: approval
-          requires:
-            - docs
-          filters:
-            branches:
-              only:
-                - master
-                - /.*all-ci.*/
-      - docs_deploy:
-          requires:
-            - docs_approve_deploy
-  # JavaScript packages that make up the SDK
-  sdk:
-    jobs:
-      - expo_sdk
-      - web_test_suite
-      # E2E SDK testing
-      - iOS-Detox
-
-  # Android and iOS clients
-  client:
-    jobs:
-      - home
-      - expotools
-      - client_android
-      - client_ios
-      - test_suite_publish
-      - android_test_suite:
-          requires:
-            - test_suite_publish
-      - client_android_approve_google_play:
-          type: approval
-          requires:
-            - client_android
-          filters:
-            branches:
-              only:
-                - /sdk-\d+/
-                - /.*all-ci.*/
-      - client_android_release_google_play:
-          requires:
-            - client_android_approve_google_play
-
-  #     - shell_app_ios_tests:
+  #     - docs_approve_deploy:
```

Comment:
restore the tests we still use. we may actually want to keep the circleci tests for the expo_sdk job around for awhile until we're confident in the transition

Reply:
figured I'd just cut down on usage while in draft